migrations/20160920122005_initial-schema.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('refresh_tokens', function (table) {
+    table.increments();
+    table.string('refresh_token', 512);
+    table.timestamps();
+  }).then(function() {
+    return knex.schema.createTableIfNotExists('access_tokens', function (table) {
```

Comment:
one with `createTable` and one with `createTableIfNotExists`?  Not sure why we need 2 separate tables... but if we really do need them is there any reason why you wouldn't do this in parallel with `Promise.all`?